src/prototype_room_creepbuilder.js

```diff
@@ -240,7 +240,7 @@ Room.prototype.applyAmount = function(input, amount) {
   if (!input) {
     return '';
   }
-  if (typeof amount === undefined) {
+  if (amount === undefined) {
```

Comment:
hum... lodash will not loop if amount not exist so i supose a simple `return output || input || ''` could allow skip input and amounts checks